runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -621,25 +620,16 @@ private void takeScreenshot(String fileName, Image image)
 		// Draw the game onto the screenshot
 		graphics.drawImage(image, gameOffsetX, gameOffsetY, null);
 
-		File playerFolder;
-		if (client.getLocalPlayer() != null && client.getLocalPlayer().getName() != null)
+		File playerFolder = SCREENSHOT_DIR;
+		if (client.getUsername() != null)
```

Comment:
You should check for Strings.isNullOrEmpty (i dont think the username can ever be null anyway)

Reply:
✔️